test/server.js

```diff
@@ -0,0 +1,17 @@
+import express from "express";
+import mockServer from "./middleware-mockserver.cjs";
+import fs from "fs";
+
+const nameHTML = fs.readFileSync( "./test/data/name.html", "utf8" );
+const app = express();
+
+app.use( mockServer() );
+
+app.post( "/test/data/name.html", function( _req, res ) {
+	res.send( nameHTML );
+} );
+app.use( express.static( "." ) );
```

Comment:
Is there a way to limit this to a few directories we need to avoid the CodeQL warning?

Reply:
I was thinking about that too. I think we can do each directory we need.